src/components/styles/Button.styled.ts

```diff
@@ -0,0 +1,27 @@
+import styled from 'styled-components'
+
+export const BaseButton = styled.div`
+  padding: 0.5rem 1rem;
+  .ant-btn-text {
+    height: auto;
+    font-size: 20px;
+    border: 0px;
+    border-bottom: transparent 2px solid;
+    background-color: #000;
+    color: #fff;
+    padding: 1px;
+    padding-bottom: 0px;
+    transition: border-bottom 0.4s ease-in-out;
+    &:hover {
+      border-bottom: 2px solid #fff;
+    }
+    &:active {
```

Comment:
I think should clarify what @hui444 meant by optional active state. I thought that she means that active is a boolean we pass as props into the button. Am ok with current implementation but being able to define active ourselves feels like more control.

Reply:
this was indicated as optional bc there are some cases where the active state is not required so do pass in a prop to make this optional

Reply:
dd62687540dc1b09686596594c98118730cff4a2